packages/blocks/src/api/serializer.js

```diff
@@ -280,9 +281,12 @@ export function serializeBlock( block ) {
  * Takes a block or set of blocks and returns the serialized post content.
  *
  * @param {Array} blocks Block(s) to serialize.
+ * @param {Object} options Serialization options.
```

Comment:
We should document these options. I think the "best" way would to define a JSDoc typedef of something like `WPBlockSerializationOptions` with each property described. Unfortunately `docgen` won't yet output these, but it seems prime for a future enhancement.